### PR TITLE
Improve translation for spanish type-hints docs

### PR DESCRIPTION
**Pull request by drcode-rune**
Fix translations and add missed example code blocks

### CODE REVIEW

**Review by SepehrRasouli — APPROVED** [review]

**Review by jovicon — APPROVED** [review]

**Comment by LeSZ0** [comment]
@tiangolo I think we can merge this PR now

**Comment by tiangolo** [comment]
📝 Docs preview for commit f76cbf1ad78639a9dccacc3bedc4cd31f0efc986 at: https://c4e3d175.fastapitiangolo.pages.dev

File: docs/es/docs/python-types.md [hunk 1]
```diff
@@ -4,23 +4,27 @@
...
 Comencemos con un ejemplo simple:
 
 ```Python
-{!../../../docs_src/python_types/tutorial001.py!}
+def get_full_name(first_name, last_name):
+    full_name = first_name.title() + " " + last_name.title()
+    return full_name
+
+
+print(get_full_name("john", "doe"))
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:27]:
```suggestion
{!../../../docs_src/python_types/tutorial001.py!}
```

Please revert these changes, the original examples should be shown.

File: docs/es/docs/python-types.md [hunk 2]
```diff
@@ -35,15 +39,16 @@ La función hace lo siguiente:
 * Convierte la primera letra de cada uno en una letra mayúscula con `title()`.
 * Las <abbr title="las junta como si fuesen una. Con el contenido de una después de la otra. En inlgés: concatenate.">concatena</abbr> con un espacio en la mitad.
 
-```Python hl_lines="2"
-{!../../../docs_src/python_types/tutorial001.py!}
+```Python
+def get_full_name(first_name, last_name):
+    full_name = first_name.title() + " " + last_name.title()
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:44]: The same here, please revert this change.

File: docs/es/docs/python-types.md [hunk 3]
```diff
@@ -65,28 +70,28 @@ Vamos a modificar una única línea de la versión previa.
 
 Vamos a cambiar exactamente este fragmento, los parámetros de la función, de:
 
-```Python
-    first_name, last_name
+```python
+first_name, last_name
 ```
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:75]: Same here.

File: docs/es/docs/python-types.md [hunk 4]
```diff
@@ -65,28 +70,28 @@ Vamos a modificar una única línea de la versión previa.
...
 
 a:
 
 ```Python
-    first_name: str, last_name: str
+first_name: str, last_name: str
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:80]: And this one

File: docs/es/docs/python-types.md [hunk 5]
```diff
@@ -237,28 +272,49 @@ Una vez más tendrás todo el soporte del editor:
 
 ## Modelos de Pydantic
 
-<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una library de Python para llevar a cabo validación de datos.
+<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una libreria de Python utilizada para llevar a cabo validación de datos.
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:275]: Please revert the word "librería", even though it's sometimes used in Spanish, it's incorrect, it should be "biblioteca", but that's not commonly used, that's why I use the technical word in English.

File: docs/es/docs/python-types.md [hunk 6]
```diff
@@ -237,28 +272,49 @@ Una vez más tendrás todo el soporte del editor:
...
 
 Tú declaras la "forma" de los datos mediante clases con atributos.
 
 Cada atributo tiene un tipo.
 
-Luego creas un instance de esa clase con algunos valores y Pydantic validará los valores, los convertirá al tipo apropiado (si ese es el caso) y te dará un objeto con todos los datos.
+Luego creas un instancia de esa clase con algunos valores y Pydantic los validará, los convertirá al tipo apropiado (si ese es el caso) y te dará un objeto con todos los datos.
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:281]: Please revert the word "instancia", even though it's sometimes used in Spanish, it's actually incorrect, "instance" means "ejemplar", that's why I keep the technical term in English.

File: docs/es/docs/python-types.md [hunk 7]
```diff
@@ -237,28 +272,49 @@ Una vez más tendrás todo el soporte del editor:
...
+}
+user = User(**external_data)
+print(user)
+# > User id=123 name='John Doe' signup_ts=datetime.datetime(2017, 6, 1, 12, 22) friends=[1, 2, 3]
+print(user.id)
+# > 123
 ```
 
-!!! info "Información"
-    Para aprender más sobre <a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic mira su documentación</a>.
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:257]: Please revert removing this info box.

File: docs/es/docs/python-types.md [hunk 8]
```diff
@@ -267,20 +323,20 @@ Vas a ver mucho más de esto en práctica en el [Tutorial - User Guide](tutorial
...
 
-Puede que todo esto suene abstracto. Pero no te preocupes que todo lo verás en acción en el [Tutorial - User Guide](tutorial/index.md){.internal-link target=_blank}.
+Puede que todo esto suene abstracto. Pero no te preocupes, todo puedes verlo en acción en el [Tutorial - Guia de usuario](tutorial/index.md).
 
-Lo importante es que usando los tipos de Python estándar en un único lugar (en vez de añadir más clases, decorator, etc.) **FastAPI** hará mucho del trabajo por ti.
+Lo importante es que usando el tipado de datos estándar de Python en un único lugar (en vez de añadir más clases, decoradores, etc.) **FastAPI** hará mucho del trabajo por ti.
 
-!!! info "Información"
-    Si ya pasaste por todo el tutorial y volviste a la sección de los tipos, una buena referencia es <a href="https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html" class="external-link" target="_blank">la "cheat sheet" de `mypy`</a>.
+**Información**
```
Comment (tiangolo) [inline docs/es/docs/python-types.md:341]: Please revert removing this info box.

**Review by tiangolo — CHANGES_REQUESTED** [review]
Thanks for the work! ☕ 

I have a few comments, several in line, but not all. Please revert all the inline code blocks, they should be included from the original examples. You can read more of the guidelines here: https://fastapi.tiangolo.com/contributing/#translations

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**